resources/views/dormitory/rooms/app.blade.php

```diff
@@ -0,0 +1,861 @@
+        $("path").hover(function(e) {
+            if(this.id.indexOf('room')>-1){
+                lastFillColor=$(this).css('fill');
+                $(this).css({'fill': '#002868', 'cursor' : 'pointer'});
+                $('#info-box').css('display','block');
+                people=$(this).data('info').split(';');
+                htmlString='';
+                
+                people.forEach(element => {
+                    if(element!=''){
+                        htmlString+='<li>'+element+'</li>';
+                    }
+                });
+                $('#info-box').html(
+                    '<p>'+this.id.replace('room','')+'</p>'+
+                    '<p>Lakók:</p>'+
+                    '<ol>'+
+                    htmlString
+                    +'</ol>'
+                );
+                // $('#info-box').html();
+            }
+        }, function(e){
+            if(this.id.indexOf('room')>-1){
+                $(this).css('fill', lastFillColor);
+                $('#info-box').css('display','none');
+                $('#info-box').html('');
+            }
+        });
+        $(document).mousemove(function(e) {
+            $('#info-box').css('top',e.pageY-$(document).scrollTop()+10);
+            $('#info-box').css('left',e.pageX+10);
+        }).mouseover();
+        $("#name").submit(function(e){
+            query=$('#nameInput').val();
+            query2=query;
+            if(query.trim().length>0 && !isNaN(query)){
+                query2='room'+query;
+                console.log('asdasdasd'); 
+            }
+            console.log(query);
+            $('#room'+query).addClass('blink');
+            $('[data-info*='+String(query2)+' i]').addClass('blink');
+            setTimeout(function(){
+                $('#room'+query).removeClass('blink');
+                $('[data-info*='+String(query2)+' i]').removeClass('blink');
+            }, 5000);
+
+        });
+    });
+    
+</script>
+@section('content')
+<div class="row">
+    <div class="col s12">
+        <div class="card">
+            <div class="card-content">
+                <div id="info-box"></div>
+                @php
+                    $colorArray=array();
+                    foreach ($rooms as $room) {
+                        $color="#ffffff";
+                        switch ($room->residentNumber()) {
+                            case 0:
+                                $color="#11f709";
+                                break;
+                            case 1:
+                                $color="#2bb505";
+                                break;
+                            case 2:
+                                $color="#f0fc0a";
+                                break;
+                            case 3:
+                                $color="#fc4f05";
+                                break;
+                            default:
+                                $color="#ffffff";
+                                break;
```

Comment:
Add the colors to the Room model, something like  in semesterstatus or workshops